tests/tasks/test_linkstatetask.py

```diff
@@ -257,13 +258,13 @@ def linkstatetask_with_admin_down(snmpsim, snmp_test_port):
         name="buick.lab.example.org", address="127.0.0.1", port=snmp_test_port, community="linksadmindown"
     )
     state = ZinoState()
-    task = LinkStateTask(device, state)
+    task = LinkStateTask(device, state, zino.state.config)
```

Comment:
Why does this task need the global config variable during tests?  Won't an empty (i.e. default values only) `Configuration` object do?